### PR TITLE
FLARE-128: Allow missing/empty study_name

### PR DESCRIPTION
**Pull request by nvidianz**

### CODE REVIEW

File: nvflare/fuel/hci/server/file_transfer.py [hunk 1]
```diff
@@ -356,7 +353,7 @@ def _validate_job(self, meta: dict, study_manager: StudyManagerSpec, fl_ctx: FLC
             required_clients = set()
             deploy_map = meta.get(JobMetaKey.DEPLOY_MAP)
             if deploy_map:
-                for k, v in deploy_map:
+                for k, v in deploy_map.items():
                     required_clients.update(v)
```
Comment (YuanTingHsieh) [inline nvflare/fuel/hci/server/file_transfer.py:357]: we can also do `required_clients.update(deploy_map.values())`

**Review by YuanTingHsieh — APPROVED** [review]

**Review by yhwen — CHANGES_REQUESTED** [review]

File: nvflare/fuel/hci/server/file_transfer.py (hunk 1 above)
Comment (YuanTingHsieh) [reply nvflare/fuel/hci/server/file_transfer.py:357]: ah.. my bad, the values are list, the current logic is good.

File: nvflare/fuel/hci/server/file_transfer.py [hunk 2]
```diff
@@ -356,7 +353,7 @@ def _validate_job(self, meta: dict, study_manager: StudyManagerSpec, fl_ctx: FLC
             required_clients = set()
```
[the remark this answers is not in the document]
Comment (nvidianz) [reply nvflare/fuel/hci/server/file_transfer.py:353]: Let's merge this and work on the logic in another PR.

[the remark this answers is not in the document]
Comment (yhwen) [reply nvflare/fuel/hci/server/file_transfer.py:353]: will address this issue in a separate PR.

**Review by yhwen — APPROVED** [review]

**Pull request merged**